### DISCUSSION TITLE
Change UI on "Leave without saving"

### DISCUSSION THREAD

**Discussion by weareturnone** [payloadcms/payload#3338] [Feature Requests & Ideas]
Can you please change the way the "Leave without saving" looks?

Our viewpoint:
- Order should be reversed, "stay on page" should be left and prioritized (black)

![CleanShot 2023-09-15 at 17 48 38@2x](https://github.com/payloadcms/payload/assets/106660686/73df7ab6-ec4a-40d0-97e7-3d0e4dd9bb2d)

**Comment by tylandavis** [comment]
@weareturnone 'Leave anyway' is the priority in this case because it confirms the user's previous action (navigating away before saving).
